src/ludamus/client/src/index.css

```
@@ -394,8 +394,49 @@ button:active {
   }
 
   .btn-primary {
-    --bg: var(--color-primary);
+    /* Bright coral face via a subtle vertical gradient — lighter top, deeper
+       bottom — both stops are --color-primary (#f85a3c) nudged in lightness
+       (no color-mix: legacy support). --bg is the bottom stop so the 3D lip
+       matches the gradient base. The white label keeps the brand's bright coral
+       in both themes; the touches below fix dark-mode halation perceptually
+       rather than by recolouring. */
+    --bg: #ef4f2d;
     color: white;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)